tutorials/0_What_is_ROS.ipynb

```diff
@@ -0,0 +1,150 @@
+    "\n",
+    "For example, let's say we have a <u>Camera node</u> which collects data from a camera and we have an <u>Image processing node</u> to process the images. This situation is depicted in the figure below. \n",
+    "Say that we want to *request* data from the Camera node at a specific time to process it using the Image processing node. In this example the Image processing node first sends requests for /image_data. The Camera node collects the data from the camera and then sends a message over /image_data as a reply to the request.   \n",
+    "\n",
+    "![](images/image_processing_example.png)\n",
+    "\n",
+    "\n",
+    "# Why ROS ?\n",
+    "ROS is certainly not the only robotics middleware available. Other options are:\n",
+    "- Urbi\n",
+    "- MIRO\n",
+    "- YARP\n",
+    "- OpenRDK\n",
+    "- etc.\n",
+    "\n",
+    "We will not go into detail on the structure and advantages of other available robotics middleware compared to ROS. We do not want to diminish their reputation or influence the reader on their choice of robotics middleware. Instead we will focus on the advantages and possibilities ROS has to offer. \n",
+    "\n",
+    "- Since nodes are connected via a distributed message system, only the required nodes need to be called which can deminish the overall program size. This modularity also has the advantage that, if a component crashes, the whole system doesn’t crash. For example, your robot can still continue its actions even if some sensors have died. \n",
+    "\n",
+    "- ROS is designed for a Linux operating system, however there is the possibility to install and use ROS on macOS and Microsoft Windows. Although these versions are experimental and will run on-top of the OS subsystem for Linux, they are supported by the ROS community. \n",
+    "\n",
+    "- ROS isn’t limited to one programming language. Nodes can be written in some of the most commonly used programming environments. You can write nodes in C++, Python, java and LISP.  Note that C++ offers the most support and that ROS works with Python 2 and is not yet converted to Python 3. \n",
+    "\n",
+    "- Through the  ROS community and the fact that ROS is open source, we can avoid the need to develop proprietary code for many aspects of robot control. Troubleshooting via the ROS community can be quick and effective. This makes the development of projects faster and easier. The tutorials on the wiki pages  help new users get comfortable with ROS, butcan sometimes be to technical or don’t explain enough. However, the community of ROS can answer all questions and help you on your way.\n",
```

Comment:
typo: butcan
typo: to technical -> too technical
Observation: We should avoid embellishing sentences. I think it should be just _can be technical_.